src/main/kotlin/ecommerce/entity/CartEntity.kt

```diff
@@ -0,0 +1,20 @@
+package ecommerce.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "cart")
+open class CartEntity(
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    val id: Long? = null,
+    @Column(name = "member_id", nullable = false)
+    val memberId: Long,
+    @Column(name = "created_at", nullable = false)
+    val createdAt: LocalDateTime = LocalDateTime.now(),
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null,
    
    @Column(name = "member_id", nullable = false)
    val memberId: Long,
    
    @Column(name = "created_at", nullable = false)
    val createdAt: LocalDateTime = LocalDateTime.now(),
```

Adding some newlines could help improve readability 👀

Reply:
Agreed! I would love to have line between some pieces of code, but our coach taught us to, from time to time, format the code using ktlinFormat (at the Gradle menu on the left side of IntelliJ), and this tool aim to clean the code even from empty lines 😐